### ISSUE TITLE
Replace file() with open() in twisted/mail

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8525]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)| @rodrigc reported|
|-|-|
|Trac ID|trac#8525|
|Type|enhancement|
|Created|2016-06-28 06:18:09Z|
|Branch|https://github.com/twisted/twisted/tree/8525-rodrigc-file|

In Python 3, file() is gone (see https://docs.python.org/3/whatsnew/3.0.html#builtins ), and needs to be replaced with open()
which works on Python 2 and 3

<details><summary>Searchable metadata</summary>

```
trac-id__8525 8525
type__enhancement enhancement
reporter__rodrigc rodrigc
priority__normal normal
milestone__Python_3_x Python-3.x
branch__8525_rodrigc_file 8525-rodrigc-file
branch_author__ 
status__closed closed
resolution__fixed fixed
component__mail mail
keywords__None None
time__1467094689297117 1467094689297117
changetime__1467114817810867 1467114817810867
version__None None
owner__rodrigc rodrigc

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_1"></a>|@rodrigc commented|
|-|-|

See:
https://github.com/twisted/twisted/pull/249

**Comment by twisted-trac** [comment]
|[<img alt="Lukasa's avatar" src="https://avatars.githubusercontent.com/u/1382556?s=50" width="50" height="50">](https://github.com/Lukasa)<a name="note_2"></a>|@Lukasa set owner to @rodrigc|
|-|-|

Looks good to me, merge when ready.

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_3"></a>|GitHub set status to `closed`|
|-|-|

In changeset 3128a3977d3ec597c47ac4166c8fb445bb15e894
```
#!CommitTicketReference repository="" revision="3128a3977d3ec597c47ac4166c8fb445bb15e894"
Merge 8525-rodrigc-file: Replace file() with open() in twisted/mail

Author: rodrigc
Reviewer: Lukasa
Fixes: #8525
```
